js/currencycom.js

```diff
@@ -1165,6 +1167,56 @@ module.exports = class currencycom extends Exchange {
         return this.parseTrades (response, market, since, limit);
     }
 
+    async fetchLeverage (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+        };
+        const response = await this.privateGetV2LeverageSettings (this.extend (request, params));
+        //
+        // {
+        //     "values": [ 1, 2, 5, 10, ],
+        //     "value": "10",
+        // }
+        //
+        return this.safeNumber (response, 'value');
+    }
+
+    async fetchLeverageTiers (symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchLeverageTiers() requires a symbol argument');
+        }
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+        };
+        const response = await this.privateGetV2LeverageSettings (this.extend (request, params));
+        //
+        // {
+        //     "values": [ 1, 2, 5, 10, ],
+        //     "value": "10",
+        // }
+        //
+        const data = this.safeValue (response, 'values');
+        const result = {};
+        result[symbol] = [];
+        for (let i = 0; i < data.length; i++) {
+            const value = data[i];
+            result[symbol].push ({
+                'tier': undefined,
```

Comment:
Should be `'tier': i` here.

Reply:
I think I might be wrong at all and that method shouldn't have been implemented. the account tiers are not returned from that endpoint (tiers are not related at all), those leverages (1,2,5,10,50,100) are just the selectable leverage amount (they doesn't have in-between slider, to set leverage to i.e. 17 or 42). So, should I remove this method completely?

Reply:
Yep, looks like it.